README.md

```diff
@@ -69,33 +69,73 @@ It is required to setup an AWS account and access the AWS IoT Console for runnin
 
 1. Go to the `build/bin` directory and run any demo executables from there.
 
-  
+### Alternative option of Docker containers for running demos locally.
```

Comment:
I think the header level got flipped with the next sections.

Reply:
Fixed